repo/packages/M/mariadb/3/package.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
Service doesnt install. Please create an example for this package here: https://github.com/dcos/examples